inspirehep/orcid/putcode_getter.py

```diff
@@ -96,10 +96,10 @@ def _get_putcodes_and_recids_iter(self, putcodes):
             if INSPIRE_WORK_URL_REGEX.match(url):
                 recid = PidStoreBase.get_pid_from_record_uri(url)[1]
                 if not recid:
-                    logger.error(
-                        "OrcidPutcodeGetter: cannot parse recid from url={} for orcid={}".format(
-                            url, self.orcid
-                        )
+                    LOGGER.error(
```

Comment:
This also looks part of the normal flow, I would send it to Sentry, wdyt?

Reply:
I believe that we will lose this record if we at least won't log it in sentry, as there are no retries.